R/transform-code.R

```diff
@@ -60,16 +98,16 @@ separate_chunks <- function(lines) {
   lst(r_chunks, text_chunks)
 }
 
-#' Identifies raw R code chunks
+#' Identifies raw Rmd code chunks
 #'
 #' Raw in the sense that these chunks don't contain pure R code, but they
 #' contain a header and footer of markdown. Only code chunks that have an engine
 #' whose name matches `engine-pattern` are considered as R code.
 #' @inheritParams separate_chunks
 #' @param engine_pattern A regular expression that must match the engine name.
 #' @keywords internal
-identify_r_raw_chunks <- function(lines, engine_pattern = get_engine_pattern()) {
-  pattern <- get_knitr_pattern(lines)
```

Comment:
We should remove the useless argument for this function.

Reply:
Wasn't 100% sure what you were referring to here but I think it might be redundant after the change I'm pushing now.

Reply:
The argument  `lines` I think.

Reply:
Oh right. Another weird code legacy 🤣

Reply:
Indeed.